docs/section19/19-26.md

```diff
@@ -0,0 +1,99 @@
+# Section 19 - Rule 19-26              
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-26               
+**Rule Description:**  For cooling sizing runs, schedules for internal loads, including those used for infiltration, occupants, lighting, gas and electricity using equipment, shall be equal to the highest hourly value used in the annual simulation runs and applied to the entire design day. For heating sizing runs, schedules for internal loads, including those used for occupants, lighting, gas and electricity using equipment, shall be equal to the lowest hourly value used in the annual simulation runs, and schedules for infiltration shall be equal to the highest hourly value used in the annual simulation runs and applied to the entire design day.
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE     
+**Appendix G Section:** G3.1.2.2.1 excluding exception         
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each Space  
+
+**Applicability Checks:**  
+1. Does not apply to residential dwelling units cooling design schedules. 
+  
+   
+**Function Calls:**  None  
+
+
+## Rule Logic:   
+- For each space in the B_RMI: `for space in B_RMI...Space:`  
+    - Reset is_dwelling_unit boolean variable: `is_dwelling_unit = false`  
+    - Check if the lighting space type is of a residential dwelling unit type, if not then set is_dwelling_unit = false: `if space.lighting_space_type in ["DWELLING_UNIT"] or space._ventilation_space_type in ["TRANSIENT_RESIDENTIAL_DWELLING_UNIT"]: is_dwelling_unit = true`  
+    
+    - Get the zone that the space is part of start by looping through zones: `for zone in B_RMI.Zone:`  
```

Comment:
What if zone has both dwelling units and non-dwelling unit spaces?

Reply:
Discuss with the group.

Reply:
In the message and also as part of the logic if any spaces are dwelling units then applied this rule.

Reply:
I am not sure why a note would be needed since the RDS checks each space?